Updating cluster files to fix Pantheon (#6706)
open-cluster-management/rhacm-docs: clusters/cluster_mce_overview.adoc

The {mce-short} is the cluster lifecycle operator that provides cluster management capabilities for {ocp-short} and {product-title-short} hub clusters. From the hub cluster, you can create and manage clusters, as well as destroy any clusters that you created. You can also hibernate, resume, and detach clusters. Learn more about the cluster lifecycle capabilities from the following documentation.

//this is broken, not sure why bc i updated the conref. After we published, that was my first time noticing the conref. i think we should use the absolute URL instead of conref
//https://access.redhat.com/articles/7073030

> Access the link:{support-matrix-mce}[Support matrix] to learn about hub cluster and managed cluster requirements and support.

Reviewer: ```suggestion
Access the link:https://access.redhat.com/articles/7073030[Support matrix] to learn about hub cluster and managed cluster requirements and support.
```

- Release images are the version of {ocp-short} that you use when you create a cluster. For clusters that are created using {product-title-short}, you can enable automatic upgrading of your release images. For more information about release images in {product-title-short}, see xref:../clusters/cluster_lifecycle/release_image_intro.adoc#release-images-intro[Release images].
//we might need to add an extra directory app ../ or change to link: I encourage us to change this to link:
//apis and troubleshooting-mce| MJ|07/23

* xref:about/mce_intro.adoc#mce-intro[About cluster lifecycle with {mce-short}]
* xref:release_notes/release_notes_intro.adoc#mce-release-notes[Release notes]
* xref:install_upgrade/install_intro.adoc#mce-install-intro[Installing and upgrading {mce-short}]
* xref:credentials/credential_intro.adoc#credentials[Managing credentials]
* xref:cluster_lifecycle/cluster_lifecycle_intro.adoc#cluster-intro[Cluster lifecycle introduction]

> * xref:../discovery/discovery_intro.adoc#discovery-intro[Discovery service introduction]
> * xref:../hosted_control_planes/hosted_intro.adoc#hosted-control-planes-intro[Hosted control planes]
> * link:../api/api_intro.adoc#apis[APIs]
> * link:../support_troubleshooting/troubleshooting_mce_intro.adoc#troubleshooting-mce[Troubleshooting]

Author: ```suggestion
* xref:discovery/discovery_intro.adoc#discovery-intro[Discovery service introduction]
* xref:hosted_control_planes/hosted_intro.adoc#hosted-control-planes-intro[Hosted control planes]
* link:api/api_intro.adoc#apis[APIs]
* link:support_troubleshooting/troubleshooting_mce_intro.adoc#troubleshooting-mce[Troubleshooting]
```

* xref:cluster_lifecycle/cluster_lifecycle_intro.adoc#cluster-intro[Cluster lifecycle introduction]
* xref:discovery/discovery_intro.adoc#discovery-intro[Discovery service introduction]
* xref:hosted_control_planes/hosted_intro.adoc#hosted-control-planes-intro[Hosted control planes]

> * link:api/api_intro.adoc#apis[APIs]

Reviewer: ```suggestion
* xref:api/api_intro.adoc#apis[APIs]
```

> * link:support_troubleshooting/troubleshooting_mce_intro.adoc#troubleshooting-mce[Troubleshooting]

Reviewer: ```suggestion
* xref:support_troubleshooting/troubleshooting_mce_intro.adoc#troubleshooting-mce[Troubleshooting]
```
